src/core/ext/transport/cronet/transport/cronet_transport.c

```diff
@@ -151,6 +151,11 @@ struct op_state {
   bool state_callback_received[OP_NUM_OPS];
   bool fail_state;
   bool flush_read;
+#ifdef GRPC_CRONET_WITH_PACKET_COALESCING
```

Comment:
In general it's better to avoid this style of feature macro: only one branch ever gets really tested, and logic gets more and more convoluted around it.